packages/@aws-cdk/aws-appmesh/lib/route-spec.ts

```diff
@@ -34,6 +34,223 @@ export interface HttpRouteMatch {
    * and you want the route to match requests to my-service.local/metrics, your prefix should be /metrics.
    */
   readonly prefixPath: string;
+
+  /**
+   * Specifies the client request headers to match on. All specified headers
+   * must match for the route to match.
+   *
+   * @default - do not match on headers
+   */
+  readonly headers?: HeaderMatch[];
+
+  /**
+   * The client request method to match on. Specify only one.
+   *
+   * @default - do not match on request method
+   */
+  readonly method?: HttpRouteMatchMethod;
+
+  /**
+   * The client request scheme to match on. Specify only one. Applicable only for HTTP2 routes.
+   *
+   * @default - do not match on HTTP2 scheme
+   */
+  readonly scheme?: HttpRouteMatchScheme;
```

Comment:
"Scheme" is a weird word to use here. How about "protocol" instead?

Reply:
Yeah. It's just the IETF's choice of wording in the HTTP/2 and URI RFCs. It corresponds directly with the [`:scheme`](https://tools.ietf.org/html/rfc7540#section-8.1.2.3) pseudo-header which refers to the request's [scheme URI portion](https://tools.ietf.org/html/rfc3986#section-3.1). Notwithstanding, I'll push up a commit that changes `scheme` -> `protocol`. It's enough to change back if there's any desire.